policy/file/source.go

```diff
@@ -348,3 +351,23 @@ func (s *Source) getFilePolicyID(file, name string) policy.PolicyID {
 func md5Sum(i interface{}) [16]byte {
 	return md5.Sum([]byte(fmt.Sprintf("%v", i)))
 }
+
+func (s *Source) GetLatestVersion(_ context.Context, policyID policy.PolicyID) (*sdk.ScalingPolicy, error) {
+	s.policyMapLock.Lock()
+	defer s.policyMapLock.Unlock()
+
+	val, ok := s.policyMap[policyID]
+	if !ok {
+		return nil, fmt.Errorf("failed to get policy %s", policyID)
```

Comment:
Nitpick on this error message: this seems like a case where the policy is no longer being watched, right?

Reply:
Yes! Lest update it.